tic_tac_toe.js

```diff
@@ -0,0 +1,92 @@
+$(document).ready(function() {
+    function Cell(el) {
+        this.move = null;
+        this.el = el;
+        this.clicked = false;
+    }
+    Cell.prototype.handleCellClick = function(rowNum, colNum, board) {
+        if (!board.table[rowNum][colNum].clicked && !board.gameOver) {
+            board.table[rowNum][colNum].clicked = true;
+            board.table[rowNum][colNum].move = board.turn;
+            this.el.html(board.turn);
+            var turn = "O";
+            if (board.turn === turn) { turn = "X"; }
+            board.turn = turn;
+            board.totalMoves++;
+            if (board.totalMoves > 4) {
```

Comment:
I like that you wait to check for a winner until it's actually possible to have one!